src/main/java/de/orchound/gameoflife/model/Board.java

```diff
@@ -59,11 +59,13 @@ public boolean getCellStatus(int rowIndex, int cellIndex) {
 	}
 
 	public void resurrectCell(int rowIndex, int cellIndex) {
-		target[rowIndex * size.x() + cellIndex] = true;
+		if (cellIndex >= 0 & cellIndex < size.x())
```

Comment:
`&` vs `&&`
We also don't have a safe-guard like this for `getCellStatus()`. I think since the board dimensions are known to the client, it should be the client's responsibility to make sure to pass the correct indices.
So move these guards to `Game`.